github_actions/spec/fixtures/vcr_cassettes/Dependabot_GithubActions_UpdateChecker/_latest_version/with_a_dependency_that_has_a_latest_requirement_and_a_valid_version/1_3_5_1.yml

```diff
@@ -0,0 +1,91 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://github.com/actions/create-release.git/info/refs?service=git-upload-pack
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      User-Agent:
+      - dependabot-core/0.154.1 excon/0.82.0 ruby/2.7.3 (x86_64-darwin19) (+https://github.com/dependabot/dependabot-core)
+      Authorization:
+      - Basic eC1hY2Nlc3MtdG9rZW46Z2hwX1ZveHh3TVZVWDBCMXZQdTdzd2N5eDEwVWxsbndwQjJhTmVzTA==
```

Comment:
```bash
echo eC1hY2Nlc3MtdG9rZW46Z2hwX1ZveHh3TVZVWDBCMXZQdTdzd2N5eDEwVWxsbndwQjJhTmVzTA== | base64 -D
```

I think we should remove any non-essential headers. The request headers aren't really necessary and the only response header that we need is probably `Content-Type`.